modules/api-gateway/src/iam-policy.json.tpl

```diff
@@ -0,0 +1,29 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": "apigateway:GET",
+      "Resource": [
+        "arn:aws:apigateway:*::/account",
```

Comment:
please have region  also included here,
use data to identify current one and pass to template,
Have an option/variable to disable iam user creation (by default set to create user), so that in case if we do not want to create user for api-gateway we will be able to disable